sentry_sdk/scope.py

```diff
@@ -512,15 +512,15 @@ def get_traceparent(self, *args, **kwargs):
             return traceparent
 
         # Fall back to isolation scope's traceparent. It always has one
-        return Scope.get_isolation_scope().get_traceparent()
+        return self.get_isolation_scope().get_traceparent()
```

Comment:
In `scope.py` here we use `self.get_x()` because in the future we will have a check in `api.get_x()` for otel or sentry and we do not want to call the check all the time?

Reply:
yes, there will be a `PotelScope` class that will inherit from `Scope` but the internal getters/setters will be different